domain/schema/controller.go

```diff
@@ -383,7 +383,8 @@ CREATE TABLE model_metadata (
     cloud_uuid            TEXT NOT NULL,
     cloud_region_uuid     TEXT,
     cloud_credential_uuid TEXT,
-    model_type_id         INT,
+    secret_backend_uuid   TEXT,
```

Comment:
This implies that every model will have a secret backend, is that true?

Reply:
Yes, you are right.